tasks/steps/blocks/extend_systemd_unit_files.yml

```diff
@@ -0,0 +1,31 @@
+---
+
+- name: 'Create directory for systemd unit files extensions'
+  file:
+    state: '{{ "directory" if cartridge_extra_env else "absent" }}'
+    recurse: '{{ true if cartridge_extra_env else false }}'
+    path: '{{ (cartridge_systemd_dir, instance_info.systemd_service_dir) | cartridge_path_join }}'
+  register: systemd_service_dir_res
+
+- name: 'Extend systemd unit files with environment'
+  copy:
+    dest: '{{ (cartridge_systemd_dir, instance_info.systemd_service_env_file) | cartridge_path_join }}'
+    content: |
+      [Service]
+      {% for key, value in cartridge_extra_env.items() %}
+      Environment={{ key }}={{ value }}
```

Comment:
Does it work fine if option contains spaces or some special characters? Don't you need some escaping?

Reply:
No, but i fixed it and added in tests:
https://github.com/tarantool/ansible-cartridge/blob/3083d473becf787581795f26fa72c8fa057f8dc9/molecule/custom_env/converge.yml#L55-L58